docs/en/connector-v2/sink/Doris.md

```diff
@@ -32,7 +32,7 @@ Version Supported
 
 ## Sink Options
 
-| Name                | Type   | Required | Default    | Description                                                                                                                                                                                                                                                                                  |
+|        Name         |  Type  | Required |  Default   |                                                                                                                                         Description                                                                                                                                          |
 |---------------------|--------|----------|------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
Revert?

Reply:
spotless will format both documents

Reply:
spotless will format this document! OK,Revert it

Reply:
merged